patches/ios-chrome-browser-https_upgrades-model-https_only_mode_upgrade_tab_helper.mm.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/ios/chrome/browser/https_upgrades/model/https_only_mode_upgrade_tab_helper.mm b/ios/chrome/browser/https_upgrades/model/https_only_mode_upgrade_tab_helper.mm
+index 052275cb0b38783f07f1ba05ce9f801178223f9c..b21b3774fff31d6a1fb8cccabf6c20c3e59b1ac5 100644
+--- a/ios/chrome/browser/https_upgrades/model/https_only_mode_upgrade_tab_helper.mm
++++ b/ios/chrome/browser/https_upgrades/model/https_only_mode_upgrade_tab_helper.mm
+@@ -270,7 +270,7 @@ void HttpsOnlyModeUpgradeTabHelper::ShouldAllowResponse(
+   // Upgrade to HTTPS if the navigation wasn't upgraded before. Ignore POST
+   // navigations.
+   if (item_pending->GetHttpsUpgradeType() == web::HttpsUpgradeType::kNone &&
+-      !navigation_is_post_) {
++      !navigation_is_post_ && !(navigation_transition_type_ & ui::PAGE_TRANSITION_FORWARD_BACK)) {
```

Comment:
please use `BRAVE_*` here so comments can be added explaining what we're doing (and also add comments :) 
It also makes it easier to see that we made a change

Reply:
fixed in https://github.com/brave/brave-core/pull/26957/commits/ec3db3f79a25a5a762215dadb6bd0fd7381427e6